src/lamp_py/ingestion/convert_gtfs_rt.py

```diff
@@ -85,16 +100,15 @@ def __init__(
         else:
             raise NoImplException(f"No Specialization for {config_type}")
 
-        self.pq_batch_size = 1024 * 256
         self.tmp_folder = "/tmp/gtfs-rt-continuous"
 
-        self.table_groups: Dict[datetime, TableData] = {}
+        self.data_parts: Dict[datetime, TableData] = {}
 
         self.error_files: List[str] = []
         self.archive_files: List[str] = []
 
     def convert(self) -> None:
-        max_tables_to_convert = 6
+        max_tables_to_convert = 50
         process_logger = ProcessLogger(
```

Comment:
will convert up to 50 million rows in one convert cycle 50 * 1_000_000 yield limit